src/SimpleSAML/Metadata/SAMLBuilder.php

```diff
@@ -173,157 +173,66 @@ public function addSecurityTokenServiceType(array $metadata): void
      */
     private function addExtensions(Configuration $metadata, RoleDescriptor $e): void
     {
+        $extensions = [];
+
         if ($metadata->hasValue('hint.cidr')) {
```

Comment:
Maybe we should just drop the _undocumented_ `hint.cidr` option and instead only keep the DiscoHints IPHint option which does the same thing?